browser/brave_wallet/brave_wallet_service_unittest.cc

```diff
@@ -3172,4 +3172,30 @@ TEST_F(BraveWalletServiceUnitTest, GenerateReceiveAddress_Btc) {
   testing::Mock::VerifyAndClearExpectations(&callback);
 }
 
+TEST_F(BraveWalletServiceUnitTest, GetAnkrSupportedChainIds) {
+  service_->GetAnkrSupportedChainIds(
+      base::BindLambdaForTesting([](const std::vector<std::string>& chains) {
+        std::vector<std::string> expected_chains = {
+            mojom::kArbitrumMainnetChainId,
+            mojom::kAvalancheMainnetChainId,
+            mojom::kBaseMainnetChainId,
+            mojom::kBinanceSmartChainMainnetChainId,
+            mojom::kMainnetChainId,
+            mojom::kFantomMainnetChainId,
+            mojom::kFlareMainnetChainId,
+            mojom::kGnosisChainId,
+            mojom::kOptimismMainnetChainId,
+            mojom::kPolygonMainnetChainId,
+            mojom::kPolygonZKEVMChainId,
+            mojom::kRolluxMainnetChainId,
+            mojom::kSyscoinMainnetChainId,
+            mojom::kZkSyncEraChainId,
+            mojom::kGoerliChainId};
+        EXPECT_EQ(chains.size(), expected_chains.size());
+        for (const auto& chain : chains) {
+          EXPECT_TRUE(base::Contains(expected_chains, chain));
+        }
+      }));
```

Comment:
nit: `EXPECT_THAT(chains, testing::UnorderedElementsAreArray(expected_chains));`
or `ElementsAreArray` if we need ordering

Reply:
https://github.com/brave/brave-core/pull/20342/commits/dbb5388879448a30cc2daaadbcc50854bd937e00